nautobot/dcim/migrations/0030_migrate_region_and_site_data_to_locations.py

```diff
@@ -191,39 +228,102 @@ def migrate_site_and_region_data_to_locations(apps, schema_editor):
             cc.locations.add(*region_locs)
             cc.save()
 
+        CustomField = apps.get_model("extras", "customfield")
+        custom_fields = CustomField.objects.filter(content_types__in=[region_ct])
+        for cf in custom_fields:
+            cf.content_types.add(location_ct)
+            cf.save()
+
+        region_locs = Location.objects.filter(location_type=region_lt).exclude(name="Global Region")
+        for location in region_locs:
+            region = Region.objects.get(name=location.name)
+            location._custom_field_data = region._custom_field_data
+        Location.objects.bulk_update(region_locs, ["_custom_field_data"], 1000)
+
+        # Custom Links do not have Region ContentType as one of its ContentType options
+
+        ExportTemplate = apps.get_model("extras", "exporttemplate")
+        export_templates = ExportTemplate.objects.filter(content_type=region_ct)
+        for et in export_templates:
+            et.content_type = location_ct
+        ExportTemplate.objects.bulk_update(export_templates, ["content_type"], 1000)
+
+        ImageAttachment = apps.get_model("extras", "imageattachment")
+        image_attachments = ImageAttachment.objects.filter(content_type=region_ct)
+        for ia in image_attachments:
+            ia.content_type = location_ct
+        ImageAttachment.objects.bulk_update(image_attachments, ["content_type"], 1000)
```

Comment:
`ImageAttachment.content_type` is part of a GenericForeignKey - these need to be updated to actually point to the appropriate Location objects.